Assets/MRTK/SDK/Features/UX/Scripts/Pointers/SpherePointer.cs

```diff
@@ -316,6 +419,22 @@ public bool TryUpdateQueryBufferForLayerMask(LayerMask layerMask, Vector3 pointe
                             }
                         }
 
+                        Vector3 closestPointToCollider = collider.ClosestPoint(pointerPosition);
+                        Vector3 relativeColliderPosition = closestPointToCollider - pointerPosition;
+
+                        // Check if the collider is within the activation cone
+                        float queryAngleRadians = queryAngle * Mathf.Deg2Rad;
+                        bool inAngle = Vector3.Dot(pointerAxis, relativeColliderPosition.normalized) >= Mathf.Cos(queryAngleRadians) || queryAngle >= 180.0f;
```

Comment:
I'm surprised to see `Vector3.Dot(pointerAxis, relativeColliderPosition.normalized) >= Mathf.Cos(queryAngleRadians)` and not `Vector3.Dot(pointerAxis, relativeColliderPosition.normalized) <= Mathf.Cos(queryAngleRadians)`

If "inangle" means "point is within the cone angle" then shouldn't the angle between axis and collider position need to be less than the query angle? Or maybe I'm missing something.

Also would be good to check that pointerAxis is mangitude 1 here.

Reply:
if the Cos(angle) is larger that means the angle itself is smaller so its the right check here.